flake.nix

```
@@ -117,6 +117,11 @@
                 xxd
               ];
               buildPhase = ''
+                if [ "$(head -c 7 ./platform/drivers/usb/usb_dev/lib/libusbdev.a)" = "version" ] ; then
+                  2>&1 echo "ERROR: Wrong binary contents; please make sure Git LFS is enabled and binaries have been pulled"
+                  exit 1
+                fi
+
```

Probably better off in the `postUnpack` phase, yeah? Also edited to not hard-code any file paths.
```suggestion
              postUnpack = ''
                if [ "$(head -c 7 $(git lfs ls-files --name-only | head -1))" =! "version" ]; then
                  echo "ERROR: run 'git lfs pull' to resolve binary files"
                  exit 1
                fi
              '';
              buildPhase = ''
```